### ISSUE TITLE
[TYPO] Le chevalier valeureux et le paladin lumineux sont mal accordés au pluriel dans le mini-event IOP

### ISSUE THREAD

**Issue by Nargacaura** [Crownicles/Crownicles#1895]
### Description du bug.

Quand on rencontre un chevalier valeureux ou un paladin lumineux, on devait avoir cette phrase:
> (...) qui fait partie des **Chevaliers valeureux**/**Paladins lumineux**. (...)

Or actuellement, on a "Chevalier valeureuxs"/"Paladin lumineuxs".

*Note: le label est à changer en "typo", vu que ce n'est pas un bug en soi.*

### Selon vous, quelle est l'importance de ce bug ?

🙂 Faible

### Reproduction du bug.

Rencontrer un chevalier valeureux ou un paladin lumineux, avec la phrase spécifique triggerant cette typo (cf screenshot).

### Captures d'écran

![Screenshot_20230308-2144192](https://user-images.githubusercontent.com/94372084/228606530-4644d0da-ce77-400f-84ee-27734b309e0e.png)

### Sur quel shard avez vous rencontré le bug ?

Tous

**Comment by leuhvraiminoryo** [comment]
ah yes tenter de rajouter une marque de pluriel à quelque chose d'invariable

**Comment by BastLast** [comment]
déplacé à après le pve

**Comment by romain22222** [comment]
j'ai regardé un peu le bug : le top serait de le fix lors de la v5. En effet, en v5, les textes ne seront pas enregistrés en bdd, ce qui nous permettra facilement de rajouter un "frenchPluralMark" dans chaque json de classe. Si on le fait avant la v5, va falloir faire une migration pour rajouter la colonne, ce qui peut être un peu relou juste pour fixer 2 typos pas très très communes

**Comment by BastLast** [comment]
> j'ai regardé un peu le bug : le top serait de le fix lors de la v5. En effet, en v5, les textes ne seront pas enregistrés en bdd, ce qui nous permettra facilement de rajouter un "frenchPluralMark" dans chaque json de classe. Si on le fait avant la v5, va falloir faire une migration pour rajouter la colonne, ce qui peut être un peu relou juste pour fixer 2 typos pas très très communes

clairement je valide, go le faire en 5.x

**Comment by BastLast** [comment]
toujours le cas en v5 ca ?  @BananePlantain  @Nargacaura  ?

**Comment by Nargacaura** [comment]
> toujours le cas en v5 ca ? [@BananePlantain](https://github.com/BananePlantain) [@Nargacaura](https://github.com/Nargacaura) ?

Yep, si je prends bien en compte que `{{class}}` est la classe au singulier.

https://github.com/Crownicles/Crownicles/blob/d027475e8f73e3fc5bc4e24204b4fb21c4eade72/Lang/fr/smallEvents.json#L269

**Comment by BastLast** [comment]
testé et fonctionne
